cgi-bin/LJ/Support.pm

```diff
@@ -1152,15 +1153,39 @@ sub work {
         }->{$rtype};
 
         # build body
-        $body = LJ::Lang::ml( "support.email.notif.update.body3", {
+        $body = LJ::Lang::ml( "support.email.notif.update.body4", {
                 sitename => $LJ::SITENAMESHORT,
                 category => $sp->{_cat}{catname},
                 subject => $sp->{subject},
-                type => $response_type,
                 username => LJ::trim( $show_name ),
                 url => "$LJ::SITEROOT/support/see_request?id=$spid",
-                text => $resp
+                type => $response_type
             } );
+        if ( $faqid ) {
+            # need to set up $lang
+            my $lang;
+            my $u = LJ::load_userid( $posterid ) if $posterid;
+            $lang = LJ::Support::prop( $spid, 'language' )
+                if LJ::is_enabled( 'support_request_language' );
+            $lang ||= $u->prop( 'browselang' ) if $u;
+            $lang ||= $LJ::DEFAULT_LANG;
+            
+            # now actually get the FAQ
+            my $faq = LJ::Faq->load( $faqid, lang => $lang );
+            if ( $faq ) {
+                $faq->render_in_place;
+                my $faqref = $faq->question_raw . " " . "$LJ::SITEROOT/support/faqbrowse?faqid=$faqid&view=full";
```

Comment:
It's a good babydev thing, so sure! It's a nice obvious kind of cleanup that would improve the readability of the code, IMO.